atomicapp/install.py

```diff
@@ -72,7 +72,8 @@ def _copyFromContainer(self, image):
                           ''.join(random.sample(string.letters, 6)))
         logger.debug("Creating a container with name %s", name)
 
-        create = [self.docker_cli, "create", "--name", name, image, "nop"]
+        # Workaround docker bug BZ1252168 by using run instead of create
+        create = [self.docker_cli, "run", "--name", name, image, "nop"]
```

Comment:
It actually doesn't matter from a functional perspective because this container never gets used. All that matters is that the container was started. The error message is really ugly though so i'll update it to override entrypoint and update the patch.